scenes/stealth_level/stealth_level.tscn

```diff
@@ -228,7 +233,7 @@ curve = SubResource("Curve2D_46hcm")
 
 [node name="Guard7-GoingBackAndForth" parent="StealthQuest/EnemyGuards" node_paths=PackedStringArray("patrol_path") instance=ExtResource("8_lbv6i")]
 position = Vector2(4607, 990)
-velocity = Vector2(-71.1522, 70.2664)
+velocity = Vector2(-70.9997, 70.4205)
```

Comment:
I just noticed that if a guard has `move_while_in_editor` set to true, when saving the velocity it had at that moment will be stored in the scene. My bad 🤦🏾 , I will make it so it set to zero on save. Created an issue to handle it: https://github.com/endlessm/threadbare/issues/81

Reply:
Ah great, I had been manually removing this hunk from each patch but forgot here.